src/options.c

```diff
@@ -488,8 +494,8 @@ static void showUsage(void)
 {
     fputs(/* Check that everything lines up after any changes. */
         "usage:  " PACKAGE " [-bcfhimopuvz] [-a X,Y,W,H] [-C NAME] [-D DISPLAY]\n"
-        "              [-d SEC] [-e CMD] [-F FILE] [-k OPT] [-l STYLE] [-n OPTS]\n"
-        "              [-q NUM] [-S CMD] [-s OPTS] [-t NUM | GEOM] [FILE]\n",
+        "              [-d SEC] [-e CMD] [-F FILE] [-k OPT] [-l STYLE] [-M MONITOR]\n"
```

Comment:
I prefer to stick with the other options: use `NUM` for an argument that expects a number.
With the word `MONITOR` someone could get confused if they have to enter a string  like: `VGA-1`, or `LVDS1` or similar.

Reply:
Sure; done.